src/core/Table/Table.test.tsx

```diff
@@ -2214,3 +2214,20 @@ it('should render zebra striped table', () => {
     container.querySelector('.iui-table-body.iui-zebra-striping'),
   ).toBeTruthy();
 });
+
+it('should sync body horizontal scroll with header scroll', () => {
```

Comment:
FYI I wrote only half of the title and GitHub Copilot ended it and wrote the whole test.